src/mlpack/methods/ann/layer/reparametrization_impl.hpp

```diff
@@ -46,7 +46,60 @@ Reparametrization<InputDataType, OutputDataType>::Reparametrization(
         << "included." << std::endl;
   }
 }
+    
+template <typename InputDataType, typename OutputDataType>
+Reparametrization<InputDataType, OutputDataType>::Reparametrization(
+    const Reparametrization& layer) :
+    latentSize(layer.latentSize),
+    stochastic(layer.stochastic),
+    includeKl(layer.includeKl),
+    beta(layer.beta)
+{
+   //Nothing to do here     
```

Comment:
```suggestion
   // Nothing to do here.
```

Pedantic style issue, missing stop and remove trailing spaces.